### PR TITLE
Team member blurbs

### PR DESCRIPTION
**Pull request by jameshadfield**
_[(preview)](https://nextstrain-s-team-mk2-zc6vbnup.herokuapp.com/team)_

Adds the ability to add (small) blurbs for each team member. Feel free to modify styling as you see fit :)

- [x] Trevor Bedford
- [x] Richard Neher
- [x] James Hadfield
- [x] Emma Hodcroft
- [x] Thomas Sibley
- [x] John Huddleston
- [x] Ivan Aksamentov
- [x] Jover Lee
- [x] Jennifer Chang
- [x] Victor Lin
- [x] Cornelius Roemer

It would also be nice to have a small blurb for alumni describing _what_ they worked on (things get out of date if we describe where they went!). 

- [ ] Cassia Wagner
- [ ] Miguel Parades
- [ ] Kairsten Fay
- [x] Moira Zuber
- [ ] Eli Harkins
- [ ] Misja Ilcisin
- [ ] Louise Moncla
- [ ] Allison Black
- [ ] Sidney Bell
- [ ] Colin Megill
- [ ] Barney Potter
- [ ] Pavel Sagulenko
- [ ] Charlton Callender

### CODE REVIEW

File: static-site/src/components/People/teamMembers.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+export const teamMembers = {
```
Comment (jameshadfield) [inline static-site/src/components/People/teamMembers.js:1]: P.S. This is the file to modify -- please add a "blurb" key to your entry!

File: static-site/src/components/People/avatars.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+const UpDown = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Blurb = styled.div`
+  font-size: 14px;
+  text-align: justify;
```
Comment (tsibley) [inline static-site/src/components/People/avatars.jsx:69]:
```suggestion
  text-align: left;
```

Screen Shot 2022-12-21 at 11 24 02
vs.
Screen Shot 2022-12-21 at 11 23 40

Comment (tsibley) [reply static-site/src/components/People/avatars.jsx]: Made as 7200e4c given "Feel free to modify styling as you see fit :)" :-)

**Comment by trvrb** [comment]
I just added my blurb and did some styling adjustments, mainly to switch to a grid instead of staggered avatar containers. I think this is significantly neater with the text blurbs.

**Review by emmahodcroft — COMMENTED** [review]
Emma's blurb

File: static-site/src/components/People/teamMembers.js (hunk 1 above)
Comment (corneliusroemer) [reply static-site/src/components/People/teamMembers.js:1]: Done for all teamMembers now

**Review by corneliusroemer — APPROVED** [review]
I think this is ready now

**Comment by victorlin** [comment]
Looks like a test is failing, I can take a look

**Comment by victorlin** [comment]
Triggered a re-build for one final preview before merging.

**Comment by tsibley** [comment]
Somewhere along the way here the footer started looking pretty janky at both a wider width

image

and a narrower width.
image

**Comment by tsibley** [comment]
Looks likely to have been 7397d2da604f083b2bcc6efe2bdd900c5a076bb1.

**Comment by tsibley** [comment]
I've made some layout adjustments to both the footer and the team page, shown below in screenshots, and will deploy now. We can tweak further in the future as desired.

## Footer
### Wide


### Narrow-ish


## Team page
### Widest


### Narrower


### Narrowest

**Pull request merged**